addons/binding/org.openhab.binding.smappee/src/main/java/org/openhab/binding/smappee/service/ReadingsUpdate.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.openhab.binding.smappee.service;
+
+/**
+ * Interface for readings updates
+ *
+ * @author Niko Tanghe
+ */
+
+public interface ReadingsUpdate {
+    public void newState(SmappeeDeviceReading devicereadings);
```

Comment:
remove `public` in interface methods are public by definition.

Reply:
You also need to add some javadoc about what these methods do.

Reply:
ok,

removed invalidconfig()